ush/generate_FV3LAM_wflow.py

```diff
@@ -296,35 +296,35 @@ def generate_FV3LAM_wflow(ushdir, logfile: str = "log.generate_FV3LAM_wflow", de
     #
     # Copy or symlink fix files
     #
-    if SYMLINK_FIX_FILES:
-
-        log_info(
-            f"""
-            Symlinking fixed files from system directory (FIXgsm) to a subdirectory (FIXam):
-              FIXgsm = '{FIXgsm}'
-              FIXam = '{FIXam}'""",
-            verbose=verbose,
-        )
-
-        ln_vrfy(f"""-fsn '{FIXgsm}' '{FIXam}'""")
-    else:
-
-        log_info(
-            f"""
-            Copying fixed files from system directory (FIXgsm) to a subdirectory (FIXam):
-              FIXgsm = '{FIXgsm}'
-              FIXam = '{FIXam}'""",
-            verbose=verbose,
-        )
-
-        check_for_preexist_dir_file(FIXam, "delete")
-        mkdir_vrfy("-p", FIXam)
-        mkdir_vrfy("-p", os.path.join(FIXam, "fix_co2_proj"))
-
-        num_files = len(FIXgsm_FILES_TO_COPY_TO_FIXam)
-        for i in range(num_files):
-            fn = f"{FIXgsm_FILES_TO_COPY_TO_FIXam[i]}"
-            cp_vrfy(os.path.join(FIXgsm, fn), os.path.join(FIXam, fn))
+    #if SYMLINK_FIX_FILES:
```

Comment:
Why are you commenting out this part? If you set "SYMLINK_FIX_FILES: false", this part will be skipped. Doesn't this resolve your issue?

Reply:
I set  SYMLINK_FIX_FILES: false, but the job still failed because of the 2nd block "else". I have to delete the block "else".

Reply:
Got it. It doesn't seem to be the best solution, but this would be ok at this time because this branch is for production.